docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
-TODO
-
-
-### Remove DCOM support from *-Computer cmdlets [5152](https://github.com/PowerShell/PowerShell/issues/5152)
-TODO
+TODO: still open
 
 
 ### Import-Csv Should apply PSTypeNames uppon import when Type Information is present in the CSV [5134](https://github.com/PowerShell/PowerShell/issues/5134)
-TODO
+Adds PSTypeName preservation on `Import-Csv` and `ConvertFrom-Csv`.
 
 
 ### -NoTypeInformation should be default on Export-Csv [5131](https://github.com/PowerShell/PowerShell/issues/5131)
-TODO
+Sets `-NoTypreInformation` as the default for `Export-Csv` and `ConvertTo-Csv`, hides the `-NoTypeInformation` parameter switch, adds `-IncludeTypeInformation` switch to `Export-Csv` and `ConvertTo-Csv` to enable legacy behavior, and provides a terminating error when both `-NoTypeInformation` and `-IncludeTypeInformation` are supplied.
 
 
 ### allow pscustomobject to have property with empty string name [5129](https://github.com/PowerShell/PowerShell/issues/5129)
-TODO
+TODO: Not taking the change, dup of [2820](https://github.com/PowerShell/PowerShell/pull/2820)
 
 
 ### StdErr/ErrorRecord should not be modified on output [5127](https://github.com/PowerShell/PowerShell/issues/5127)
-TODO
+TODO: Closed, not merged
 
 
 ### -Verbose should not override $ErrorActionPreference [5113](https://github.com/PowerShell/PowerShell/issues/5113)
-TODO
+`$ErrorActionPreference` is longer overwritten by the `-Verbose` or `-Debug` parameters.
 
 
 ### Web Cmdlets should warn when  legacy -Credential is sent over unencrypted connections [5112](https://github.com/PowerShell/PowerShell/issues/5112)
-TODO
+An error was added when a user tries to use `-Credential` (legacy usage without `-Authentication`) or `-UseDefaultCredentials` over a non-HTTPS URI. Users can bypass the error with `-AllowUnencryptedAuthentication`.
 
 
 ### Use InvariantCulture for implicit argument conversion [5106](https://github.com/PowerShell/PowerShell/issues/5106)
-TODO
+TODO: Closed, not merged
 
 
 ### Rename powershell.exe to pwsh.exe [5101](https://github.com/PowerShell/PowerShell/issues/5101)
-TODO
+TODO: (Not sure how to summarize this change further than the title)
 
 
-### Remove '-ComputerName' from 'Get/Set/Remove-Service' [5094](https://github.com/PowerShell/PowerShell/issues/5094)
-TODO
+### Remove '-ComputerName' from *-Service cmdlets [5090](https://github.com/PowerShell/PowerShell/issues/5094)
+Although `-ComputerName` worked correctly, it was removed from `*-Service` cmdlets so that users consistently rely on PSRP and only have that port open.
 
 
 ### Unify cmdlets with parameter 'Encoding' to be of type System.Text.Encoding [5080](https://github.com/PowerShell/PowerShell/issues/5080)
-TODO
+The `-Encoding` Byte has been removed from the filesystem provider cmdlets. A new parameter -Byte is now the signal that a byte stream is required as input or output will be a stream of bytes.
```

Comment:
```none
Previously, an enum value of `Byte` was used with `-Encoding` to indicate a byte stream.
However, `Byte` is not an actual encoding.  The change is remove that enum value and replace
it with a `-Byte` switch to indicate input or output should be a byte stream.
```

Reply:
I think the parameter is `-AsByteStream`, right?